src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
+
+  if (days >= 7) {
+    sum -= 50;
+  } else if (days >= 3) {
+    return (sum - 20);
+  }
```

Comment:
avoid such nesting `else if` conditions
This code is hard to read and maintain

Reply:
what a magic numbers 🪄
lets create variables and then use them
it is better for managing them from one place and for understanding what do they mean